open_buildings/download_buildings.py

```diff
@@ -81,49 +86,53 @@ def quadkey(geojson_input):
         geojson_data = json.load(click.get_text_stream('stdin'))
     
     result = geojson_to_quadkey(geojson_data)
+    print()
     click.echo(result)
 
 @cli.command()
 @click.argument('geojson_input', type=click.File('r'), required=False)
-def WKT(geojson_input):
-    """Convert GeoJSON to Well Known Text."""
+@click.option('--geocode', type=str, is_flag=True, help='geocode using a city or region')
+def WKT(geojson_input, geocode_input):
+    """Convert GeoJSON to Well Known Text."""          
     if geojson_input:
         geojson_data = json.load(geojson_input)
     else:
-        geojson_data = json.load(click.get_text_stream('stdin'))
-    
-    result = geojson_to_wkt(geojson_data)
+        geojson_data = json.load(click.get_text_stream('stdin'))   
+    if geocode_input: 
+        result = geocode_to_wkt(geocode_input)
+    else:
+        result = geojson_to_wkt(geojson_data)
     click.echo(result)
 
 
 @click.command()
 @click.argument('geojson_input', type=click.File('r'), required=False)
```

Comment:
leaving the decorators (click options) here but hashing out the function they are attached to likely causes issues. either unhash the function again, or hash everything out including the decorators. i'd leave the whole sql function unhashed for now and deal with it in a different PR.